owncloudApp/src/main/java/com/owncloud/android/extensions/OCTransferExt.kt

```diff
@@ -0,0 +1,64 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Juan Carlos Garrote Gascón
+ *
+ * Copyright (C) 2022 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.extensions
+
+import androidx.annotation.StringRes
+import com.owncloud.android.R
+import com.owncloud.android.domain.transfers.model.OCTransfer
+import com.owncloud.android.domain.transfers.model.TransferResult
+import com.owncloud.android.domain.transfers.model.TransferStatus
+
+@StringRes
+fun OCTransfer.statusToStringRes(): Int {
+     return when (status) {
+         // Really uploading, bind the progress bar to listen for progress updates
```

Comment:
Well, it refers to the first status, `TransferStatus.TRANSFER_IN_PROGRESS`. I'd say it was there before also with no logic related to the progress bar, so just indicating that, but I'll remove it so that it doesn't lead to confusion 👍